draft-ietf-rats-corim.md

```diff
@@ -1165,7 +1165,9 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/310
+The Conditional Endorsement Series Triple is used to assert endorsed values conditional on various sets of possible measurement values.
+Each series entry describes a different possible set of values.
+Series entries are ordered so that the set describing the most trustworthy state is evaluated first and least trustworthy state last.
```

Comment:
The ordering leads to negative conditions in the head of the rule still. The possible evaluation inconsistency from Issue #321 remains.

If the conditions are mutually-exclusive, then there's no need to order them and the triple is syntactic sugar for multiple normal conditions.
If the conditions are not mutually-exclusive, then you need a way to stratify condition matching to regain logical consistency.

Given that you haven't stated any condition, I don't know what your intention is here.